conf/schema/shop.json

```diff
@@ -1477,7 +1477,8 @@
         "#": "Count in tag info is 30943 on 2015-09-01",
         "description": "A group of stores, typically associated jth a single building structure.",
 	"categories": [
-            "multiuse"
+            "multiuse",
+            "building"
         ],
```

Comment:
Eventually, we may want to make all shops buildings but saving that for another time